src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/DataGridView.DataGridViewTooltipTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Data;
+using Xunit;
+
+namespace System.Windows.Forms.Tests
+{
+    public class DataGridView_DataGridViewToolTipTests
+    {
+        [WinFormsFact]
+        public void DataGridViewToolTip_Get_Not_ThrowsException()
+        {
+            DataTable dataTable = new();
+            dataTable.Columns.Add(columnName: "name");
+            dataTable.Rows.Add(values: "name1");
+
+            // create DataGridView with DataSource set to dataTable
+            DataGridView dataGridView = new();
```

Comment:
❗ All disposable objects must be disposed
```suggestion
            using DataGridView dataGridView = new();
```